packages/cli/src/tools/assertRequiredOptions.ts

```diff
@@ -6,14 +6,22 @@
  *
  * @format
  */
-
 import {Option} from 'commander';
 import {camelCase} from 'lodash';
+import {Command} from '@react-native-community/cli-types';
+
+type Options = NonNullable<Command['options']>;
+type PassedOptions = {
+  [x: string]: unknown;
```

Comment:
Let's be descriptive where we can
```suggestion
  [key: string]: unknown;
```